README.md

```diff
@@ -34,9 +34,37 @@ docker run --rm --init --ulimit memlock=-1:-1 jarredsumner/bun:edge
 
 If using Linux, kernel version 5.6 or higher is strongly recommended, but the minimum is 5.1.
 
+### Unzip is required
```

Comment:
This is great, but could you move this to `Troubleshooting`?

Reply:
Moved section and toc entry